ui/images/globe-icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="e2a1f805-85d3-4fef-9d47-1df2fb6310ad" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21.5 21.5">
+  <title>Artboard 107</title>
```

Comment:
You can delete the `title` node, as we don't want `Artboard 107` to appear when hovering the icon